std/assembly/util/math.ts

```diff
@@ -0,0 +1,1924 @@
+/*
+ * Copyright (c) 2018, Arm Limited.
+ * SPDX-License-Identifier: MIT
+ */
```

Comment:
I guess instead of doing this, we should add the origin of this code to NOTICE, similar how we do with other 3rd party code - and then, on each function, add a reference to the original source file so someone double-checking this can compare it, similar to how we do this for musl sources.

Reply:
sure! will do

Reply:
Done!